Assets/Code/CameraManager.cs

```diff
@@ -134,24 +136,51 @@ public class CameraManager : MonoBehaviour {
         }
       }
     }
-    
+
     // ------------------------------------------------------------------------
     public void OnScenarioStarted() {
       //Move to the first ViewPoint, if there is one
       var viewPoint = _objectCircularLists.GetNext(WorldObjectType.ViewPoint);
       if (viewPoint) {
+        Debug.Log($"Moving starting camera to: {viewPoint.name}");
         MoveCameraToViewPoint(viewPoint as ViewPoint.ViewPoint);
       }
-      
-      //This will auto slave the camera target to the first user. If none, try the first computer.
-      var ub = _objectCircularLists.GetNext(WorldObjectType.User);
-      if (ub != null) {
-        MoveCameraTarget(ub.transform);
-      }
+      // If we don't have a starting viewpoint, then try moving to a user
       else {
-        var computer = _objectCircularLists.GetNext(WorldObjectType.Computer);
-        if (computer) {
-          MoveCameraTarget(computer.transform);
+        var ub = _objectCircularLists.GetNext(WorldObjectType.User);
+        if (ub != null) {
+          Debug.Log($"Moving starting camera to: {ub.name}");
+          MoveCameraTarget(ub.transform);
+        }
+        // If we don't have a user to move to, then try to find a computer
+        else {
+          var computer = _objectCircularLists.GetNext(WorldObjectType.Computer);
+          if (computer) {
+            Debug.Log($"Moving starting camera to: {computer.name}");
+            MoveCameraTarget(computer.transform);
+          }
+        }
+      }
+    }
+
+    // ------------------------------------------------------------------------
+    public void MoveCameraToUser(string serverMessage) {
+      StringReader xmlreader = new StringReader(serverMessage);
+      XmlDocument xml_doc = new XmlDocument();
+      try {
+        xml_doc.Load(xmlreader);
+      }
+      catch (XmlException  e) {
+        Debug.LogError($"cameraToUser xml contains an error: {serverMessage}. {e}");
+      }
+      
+      XmlNode mainNode = xml_doc.SelectSingleNode("//cameraToUser");
+      var username = mainNode["name"].InnerText;
+      Debug.Log($"Trying to move camera to: {username}");
+      foreach (var user in _userList.Value) {
```

Comment:
How come you're not doing a fancy `_userlist.Value.find(x => x.Data.user_name == username)` here?

Reply:
Not sure it really matters.  I'd still have to check if the result is null, so it doesn't save us any lines of code and functionally it's the same thing.